processor/schemaprocessor/internal/migrate/multi_conditional.go

```diff
@@ -0,0 +1,86 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package migrate // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/schemaprocessor/internal/migrate"
+
+import (
+	"errors"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/otel/schema/v1.0/ast"
+)
+
+type set = map[string]struct{}
+
+// MultiConditionalAttributeSet maps from string keys to possible values for each of those keys.  If a valid key is passed for each value, the conditional returns true
+type MultiConditionalAttributeSet struct {
```

Comment:
Is this used for `apply_to_spans`, `apply_to_events` and `apply_to_metrics ` transforms?

Reply:
`apply_to_events`, since its the only one with multiple conditions